rollup.js

```diff
@@ -0,0 +1,72 @@
+const rollup = require('rollup').rollup;
+const commonjs = require('rollup-plugin-commonjs');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const replace = require('rollup-plugin-replace');
+const path = require('path');
+
+const config = {
+  entry: {
+    simple: './lib/es6/src/simple/simpleRoot.js',
```

Comment:
only `simple` and `logo` work out of the box. `todomvc` needs to do something about `director`s `querystring` and `util` dependencies. `interop` probably needs some configuraiton around commonjs modules.